Activities/Scripting/UiPath.Scripting.Activities/Properties/SharedResources.cs

```diff
@@ -0,0 +1,7 @@
+﻿
+namespace UiPath.Shared.Localization
+{
+    class SharedResources : UiPath.Scripting.Activities.Properties.Resources
+    {
+    }
+}
```

Comment:
Please add new line here

Reply:
and remove the one above, the first line